### PR TITLE
Added documentation for interval with offset

### PR DESCRIPTION
**Pull request by arjank**
**Description:**
Documentation change for related pull request in home-assistant/home-assistant.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#10091

### CODE REVIEW

**Comment by frenck** [comment]
👍  @arjank, looking good! Great first contribution 🏅 
Could you please rebase it against the `next` branch?

File: source/_cookbook/automation_using_timeinterval_inputboolean.markdown [hunk 1]
```diff
@@ -40,3 +40,22 @@ automation:
       transition: 5
       brightness: 255
 ```
+
+It is also possible to change the color every fifteen minutes, starting 5 minutes after the hour. That would trigger a change at h:05, h:20, h:35, and h:50.
+
+```yaml
+automation:
+# Changes Hue light every two minutes to random color if input boolean is set to on
```
Comment (frenck) [inline source/_cookbook/automation_using_timeinterval_inputboolean.markdown]: The comment does not match the example.

**Comment by frenck** [comment]
@arjank Did spot something, could you please take a look?

File: source/_cookbook/automation_using_timeinterval_inputboolean.markdown (hunk 1 above)
Comment (arjank) [reply source/_cookbook/automation_using_timeinterval_inputboolean.markdown]: Copy-paste error. Fixed.

**Comment by amelchio** [comment]
Actually, I think the best place to document this addition would be here: https://home-assistant.io/docs/automation/trigger/#time-trigger

**Comment by arjank** [comment]
@amelchio That makes sense. I've moved the documentation and updated it somewhat to match the other examples more closely.

**Pull request closed without merging**